src/pages/bulkFactoryCreation/index.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+    <div class="text-3xl font-bold">Bulk Factory Creation</div>
+    <br />
+    <div class="grid gap-4 grid-cols-1">
+        <div v-if="!props.state.accountName">
+            <span>You are not currently logged in, please log in to view this page.</span>
+        </div>
+        <template v-else>
+            <div class="grid gap-4 grid-cols-1 text-md p-2 border rounded-md px-4 bg-neutral-700 border-neutral-600">
+                <div class="grid gap-px grid-rows-1">
+                    <span class="[font-family:'Inter-Regular',Helvetica] text-[#f9d198]">
+                        <b>Input Format</b>
+                    </span>
+                    <br />
+                    <p>
+                        Please ensure your CSV file is formatted as specified below. The first row should contain column headers,
+                        and each subsequent row should include the corresponding data for a Uniq Factory. All fields are required
+                        unless otherwise specified.
+                    </p>
+                </div>
+                <div class="grid gap-px grid-rows-1 mb-2">
+                    <p>For a detailed example, download the template: <a href="/resources/bulk-factories.csv" target="_blank">bulk-factories.csv</a>
+                    </p>
+                </div>
+            </div>
+
+            <div class="grid gap-4 grid-cols-1 text-md p-2 border rounded-md px-5 border-neutral-700">
+                <div class="grid gap-px grid-rows-1"></div>
+                <div class="grid grid-rows-1 gap-4">
+                    <div class="grid grid-cols-2 gap-4">
+                        <LabelWithTooltip label="Authorizer" />
+                        <div class="flex flex-row h-12 gap-4">
+                            <input
+                                class="flex-grow rounded bg-neutral-950 text-neutral-200 pl-4 border border-neutral-700 focus:outline-none pr-4"
+                                v-model="authorizer"
```

Comment:
For ease of use, please use similar code to airdrop tool to set the authorizer and permission values automatically. E.g.:

```
const refreshDefaultInput = () => {
    if (props.state.accountName) {
        if (!authorizer.value || authorizer.value.length === 0) {
            if (I.ELEVATED_ACCOUNTS.includes(props.state.accountName)) {
                authorizer.value = 'ultra.mrktng';
                permission.value = 'team';
            } else {
                authorizer.value = props.state.accountName;
                permission.value = 'active';
            }
        }
        return true;
    }
    return false;
};
```